FieldOpt/Optimization/case.cpp

```diff
@@ -303,6 +304,21 @@ double Case::GetEnsembleAverageOfv() const {
     return sum / ensemble_ofvs_.size();
 }
 
+QPair<double, double> Case::GetEnsembleExpectedOfv() const {
+    if (ensemble_ofvs_.size() == 1) {
+        std::cerr << "WARNING: Only one realization case was successfully evaluated. "
```

Comment:
Use the new printing type: `Printer::ext_warn("Message", "SentFromClass", "Module");`